atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1889,6 +1889,13 @@ v8::Local<v8::Value> WebContents::GetLastWebPreferences(
   return mate::ConvertToV8(isolate, *web_preferences->last_preference());
 }
 
+bool WebContents::IsRemoteModuleEnabled() const {
+  if (auto* web_preferences = WebContentsPreferences::From(web_contents())) {
```

Comment:
It is possible for `web_preferences` to be `nullptr`

Reply:
maybe, the other methods check for nullptr as well, why?

Reply:
Oh a misread this, didn't realize the assignment was inside the `if` 👍